.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+script:
+  - ./bin/runci.sh $CI_TEST
+jdk:
+ - oraclejdk8
+ env:
+   global:
+   - TRAVIS_NODE_VERSION="8"
+   matrix:
+     - CI_TEST: check
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.11.12
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.11.12
+       CI_SCALA_JS: true
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.12.4
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.12.4
+     - CI_TEST: ci
+       CI_SCALA_VERSION: 2.12.4
+       CI_SCALA_JS: true
+cache:
+  directories:
+  - $HOME/.sbt/0.13/dependency
+  - $HOME/.sbt/boot/scala*
+  - $HOME/.sbt/launchers
+  - $HOME/.ivy2/cache
+  - $HOME/.coursier
+  - target/repos
+before_cache:
+  - du -h -d 1 $HOME/.ivy2/cache
+  - du -h -d 2 $HOME/.sbt/
+  - find $HOME/.sbt -name "*.lock" -type f -delete
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -type f -delete
+  - find $HOME/.ivy2/cache -name "*scalafix*.xml" -type f -delete
```

Comment:
s/scalafix/scalagen/

Reply:
Is this stuff needed at all? I don't know why it was added to the scalameta repo in the first place.

This is from the scalameta workspace so I presume that line is not needed?

Reply:
this line is only necessary if you publishLocal and use from scripted tests, so in the case of scalagen it can be helpful.

Reply:
:+1: